UPGRADE.md

```diff
@@ -15,6 +15,17 @@ INSERT INTO `re_operators` (`id`, `operator`, `type`, `inputType`) VALUES
     (19, '!=', 6, 'auto-complete');
 ```
 
+Additionally the filter by country has changed. Run following SQL script to update your filter conditions:
+
+```sql
+UPDATE `re_conditions` SET `field` = 'countryId', `type` = 6, `value` = CONCAT('"', (SELECT `id` FROM `co_countries` WHERE `code` = REPLACE(`re_conditions`. `value`, '"', '') LIMIT 1), '"') WHERE `field` = 'countryCode' AND `operator` != 'LIKE';
```

Comment:
This query does not work for me. The value is written with quotes in the database, which does not happen when I simply add a condition using the new form.